src/components/Nav/Nav.js

```diff
@@ -1,8 +1,42 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './Nav.scss';
 
-const Nav = () => {
-  return <h1>Nav</h1>;
-};
-
-export default Nav;
+export default function Nav() {
+  return (
+    <header>
```

Comment:
```suggestion
    <header className="nav">
```
가장 최상위 부모태그에 컴포넌트 이름과 동일하게 입력해주세요!